src/libstd/sys/common/net.rs

```diff
@@ -164,6 +164,92 @@ pub fn lookup_addr(addr: &IpAddr) -> io::Result<String> {
 }
 
 ////////////////////////////////////////////////////////////////////////////////
+// Timeouts
+////////////////////////////////////////////////////////////////////////////////
+
+#[cfg(target_os = "windows")]
```

Comment:
Could you move this platform-specific code into src/libstd/sys/{unix,windows}/net.rs ?